docs/Developers-Guide.md

```diff
@@ -199,7 +199,7 @@ formats like 02/01/12.
 
 ## Code style
 
-The OTP code style is described on a separate [style guide page](Codestyle.md).
+The OTP code style is described on a separate [style guide page](../doc/dev/decisionrecords/Codestyle.md).
```

Comment:
mkdocs doesn't like it when you use a relative link from its source directory `docs` to a file outside of it, because that would not work in the deployed HTML documentation.

Besides, having both a `docs` and a `doc` folder is quite weird. Can we move the DRs into `docs`?

Reply:
We can not move DRs into `docs` now, since that would publish them. But, the goal is to merge these; hence the `doc/dev`. Next step is to move `docs` into `doc(s)/user`.  I think I prefer `doc/user` and not `docs/user` since we in general use singular form for folders/packages. I checked the Spring project and they also uses just `doc` (https://github.com/spring/spring).